### PR TITLE
Update AnVIL entity lists to match mocks. #128.

### PR DESCRIPTION
**Pull request by frano-m**
### Reviewers

#128 
@MillenniumFalconMechanic 

### Changes
- refactored Table component to match mocks (generally).
  - excludes table toolbar with edit columns button and summary toggle.
  - excludes pagination.
- added `width` to columns config.
  - specific column widths are generalized for all configured tables.
- TODO references to ticket #128 are removed, but the nTag typings will need updating with configuration of cell content.

### Not completed
- Table storybook.

### CODE REVIEW

File: explorer/app/components/Table/table.tsx [hunk 1]
```diff
@@ -7,35 +18,33 @@ import {
...
+import { CheckboxMenu, CheckboxMenuItem } from "../CheckboxMenu/checkboxMenu";
 import { newColumnKey, newColumnOrder } from "./functions";
+import { Pagination } from "../Pagination/pagination";
+
+// Styles
+import { RoundedPaper } from "../common/Paper/paper.styles";
+import { Table as GridTable, TableToolbar } from "./table.styles";
 
 export interface EditColumnConfig {
+  onVisibleColumnsChange: (newColumnId: string) => void;
```
Comment (frano-m) [inline explorer/app/components/Table/table.tsx:33]: Reordered interface

File: explorer/app/components/Table/table.tsx [hunk 2]
```diff
@@ -50,28 +59,30 @@ interface TableProps<T extends object> {
...
  * @param tableProps.pagination - Config for rendering pagination and corresponding events.
  * @param tableProps.sort - Config for rendering current sort and handling corresponding events.
+ * @param tableProps.gridTemplateColumns - Defines grid table track sizing.
  * @returns Configured table element for display.
  */
 export const Table = <T extends object>({
-  items,
   columns,
   editColumns,
+  gridTemplateColumns,
```
Comment (frano-m) [inline explorer/app/components/Table/table.tsx:68]: Reordered.

File: explorer/app/components/Table/table.tsx [hunk 3]
```diff
@@ -50,28 +59,30 @@ interface TableProps<T extends object> {
...
+  items,
   pageSize,
-  total,
   pagination,
   sort,
+  total,
 }: TableProps<T>): JSX.Element => {
   const {
+    canNextPage: tableCanNextPage,
```
Comment (frano-m) [inline explorer/app/components/Table/table.tsx:76]: Reordered.

File: explorer/app/components/TableCreator/tableCreator.tsx [hunk 4]
```diff
@@ -48,13 +80,14 @@ export const TableCreator = <T extends object>({
 
   return (
     <Table<T>
-      items={items}
       columns={reactVisibleColumns}
+      editColumns={editColumns}
```
Comment (frano-m) [inline explorer/app/components/TableCreator/tableCreator.tsx:84]: Reordered props.

File: explorer/app/config/model.ts [hunk 5]
```diff
@@ -107,8 +107,32 @@ export interface ColumnConfig<
...
+export interface GridTrackMinMax {
+  max: GridTrackAuto | GridTrackFlex | GridTrackLength;
+  min: GridTrackAuto | GridTrackLength;
+}
+
+/**
+ * A selection of possible types of track sizing values of each track (column).
+ * See https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-columns#values.
+ */
+export type GridTrackSize =
```
Comment (frano-m) [inline explorer/app/config/model.ts:130]: GridTrackSize is not an exhaustive list of options, but a good first round of possible ways to define the track size.

File: explorer/app/theme/theme.ts [hunk 6]
```diff
@@ -309,7 +309,7 @@ export const theme = createTheme(defaultTheme, {
           borderColor: defaultTheme.palette.smoke,
           borderStyle: "solid",
           borderWidth: 1,
-          boxShadow: "0px 1px 4px 0px rgba(0, 0, 0, 0.07)",
+          boxShadow: "0 1px 4px 0 rgba(0, 0, 0, 0.07)",
```
Comment (frano-m) [inline explorer/app/theme/theme.ts:312]: Required - was not rendering correctly.

**Review by MillenniumFalconMechanic — APPROVED** [review]
LGTM, thanks F!

**Pull request merged**